modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/ExpressionFactoryImpl.java

```diff
@@ -376,17 +388,45 @@ private void expandBounds(
             List<RexNode> curLower,
             List<RexNode> curUpper,
             boolean lowerInclude,
-            boolean upperInclude
+            boolean upperInclude,
+            boolean containNull
```

Comment:
as far as I an see, `containNull` is never `false`. Thus, why do we need this flag?